utils/Makefile

```diff
@@ -59,13 +59,18 @@ docs: $(MANPAGES)
 viewcover:
 	xdg-open cover/index.html
 
-virtualenv:
+# Conditional virtualenv building strategy taken from this great post
+# by Marcel Hellkamp:
+# http://blog.bottlepy.org/2012/07/16/virtualenv-and-makefiles.html
+venv: oo-installenv/bin/activate
+oo-installenv/bin/activate: test-requirements.txt
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
-	virtualenv $(NAME)env
+	test -d venv || virtualenv $(NAME)env
```

Comment:
That's true. The `test` should be testing `$(NAME)env`. I failed to update that part after I copied the example from the original source (linked in the comment block).